tests/Unit/Subscriber/Behavior/VersionSubscriberTest.php

```diff
@@ -343,6 +345,7 @@ public function testRestoreLocalizedProperties()
         $property3 = $this->prophesize(PropertyInterface::class);
         $property3->getName()->willReturn('jcr:uuid');
         $node->getProperties()->willReturn([$property1->reveal(), $property2->reveal(), $property3->reveal()]);
+        $node->getNodes()->willReturn([]);
```

Comment:
Please add also a test where the node has children, which should not be copied (as it is the case with the pages).